src/Command/SqlGeneratorCommand.php

```diff
@@ -0,0 +1,229 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace CoolBeans\Command;
+
+use Symfony\Component\Console\Input\InputArgument;
+
+class SqlGeneratorCommand extends \Symfony\Component\Console\Command\Command
+{
+    private const INDENTATION = '    ';
+    public static $defaultName = 'sqlGenerator';
+
+    public function __construct()
+    {
+        parent::__construct(self::$defaultName);
+    }
+
+    protected function configure() : void
+    {
+        $this->setName(self::$defaultName);
+        $this->setDescription('Converts Beans into SQL.');
+        $this->addArgument('destination', \Symfony\Component\Console\Input\InputArgument::REQUIRED, 'Path to folder');
```

Comment:
Destination somewhat implies output directory, it should be "source" as path to directory with Beans.